File: Dev/51900718/src/main/java/tdtu/advanced/java/thinh68/models/Phieunhap.cs
namespace ComNha.Models
{
    public class Phieunhap
    {
        public string MaPhieuNhap { get; set; } = null!;
        public DateTime NgayNhapHang { get; set; }
        public double TongGiaNhap { get; set; }
        public string GhiChu { get; set; } = null!;
        public string MaNhaCungCap { get; set; } = null!;
        public string MaNhanVien { get; set; } = null!;
    }
}

File: Dev/Models/Models/PhieuNhap.java
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Entity
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@Table(name="phieunhap")
public  class PhieuNhap {
    @Id
    @GeneratedValue
    private Long MaPhieuNhap;
    private DateTime NgayNhapHang;
    private double TongGiaNhap;
    private String GhiChu;
    private String MaNhaCungCap;
    private String MaNhanVien;
}